src/src/com/microsoft/aad/adal/BasicWebViewClient.java

```diff
@@ -187,7 +208,7 @@ public void run() {
                                         .containsKey(AuthenticationConstants.OAuth2.CLIENT_ID)) {
                                     loadUrl = loadUrl + "?" + mQueryParam;
                                 }
-                                Logger.v(TAG, "Loadurl:" + loadUrl);
+                                Logger.v(TAG, "Challenge submit url:" + challengeResponse.mSubmitUrl);
```

Comment:
this is not future proof. server may return UPN as a QP. It currently happens when AA installation is kicked off from the webview.